src/lib/application/files/js/src/main.js

```diff
@@ -5,4 +5,4 @@ async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   await app.listen(3000);
 }
-bootstrap();
+void bootstrap();
```

Comment:
using `void` like this isn't common for user-facing nodejs project. Instead, we guess we should ignore lintting errors on `application/files/**/*.{js,ts}` files

Reply:
Yeah, I don't think we should use `void` here either

Reply:
ok, removed
left only formatting issues fixes